composer/callbacks/runtime_estimator.py

```diff
@@ -64,6 +66,7 @@ def __init__(self, skip_batches: int = 1, time_unit: str = 'hours') -> None:
         self.start_dur = None
         self.train_dataloader_len = None
 
+        self.time_unit = time_unit
```

Comment:
Recommend this, or updating line 111 to only log when its not none
```suggestion
        self.time_unit = time_unit or 'hours'
```

Reply:
Oh wait I'm dumb, `time_unit: str = 'hours'` should be good enough above 😛